Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -479,6 +488,21 @@ public static DartAnalysisServerService getInstance() {
     }
   }
 
+  /**
+   * Returns {@link OverrideMember}s for the given file.
+   * Empty, but not null.
+   */
+  @NotNull
+  public List<OverrideMember> getOverrideMembers(@NotNull final VirtualFile file) {
+    synchronized (myOverrideData) {
+      List<OverrideMember> regions = myOverrideData.get(file.getPath());
```

Comment:
I will update `computedOverrides` instead, as we do for all the other notifications.